pkgs/development/libraries/libprom/default.nix

```diff
@@ -0,0 +1,39 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
```

Comment:
Why is this necessary? I tested with the default `stdenv`, and it built fine.

Reply:
So it builds on Mac.

Reply:
Understood. Maybe add a comment for that as well, then, please :)